wallet-new/server/Cardano/Wallet/API/V1/Swagger.hs

```diff
@@ -503,6 +583,404 @@ Disable TLS (Not Recommended)
 -----------------------------
 
 If needed, you can disable TLS by providing the `--no-tls` flag to the wallet or by running a wallet in debug mode with `--wallet-debug` turned on.
+
+
+Common Use-Cases
+================
+
+Sending Money to Multiple Recipients
+------------------------------------
+
+As seen in [Sending ADA](#section/Getting-Started/Sending-ADA), you can send _ADA_ to
+another party using the [`POST /api/v1/transactions`](#tag/Transactions%2Fpaths%2F~1api~1v1~1transactions%2Fpost) endpoint.
+Important to notice is the type of the field `destinations`: it's a list, enabling you to provide more
+than one destination. Each destination is composed of:
```

Comment:
hmmm, I think I would rewrite as "Note the type of the field `destinations`: it is a list, enabling you to provide more than one destination."